docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -221,7 +221,7 @@ Step 2: Disk Formatting
 
 #. Create the boot pool::
 
-     zpool create \
+     zpool create -f \
```

Comment:
-f Forces use of vdevs, even if they appear in use or specify a conflicting replication level.  Not all devices can be overridden in this manner.
You don't want garbaged filesystem, just make clear, that you don't use this partition

Reply:
+1, it's very dangerous!

Reply:
Ok I made my installations on _used disks_ and one with old pools. This is probably the reason the system ask me to force pool creation.
I'll roll back these, but can you suggest me some documentations explain why a _forced pool creation_ can lead to a _garbaged filesystem_?
Why a _forced creation_ cannot be a good start?
Thx

Reply:
You have 2 proccesses which ones can write to this block device. It's the same if you use iSCSI share on two systems with not distributed Filesystem on it and write some data on it on both systems. Just as a result you can have /etc/fstab as character device or something weird like that.